<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>311</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>280</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/toolbaricons/icons/advanced.png</normaloff>:/toolbaricons/icons/advanced.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="connectionSettingTab">
      <attribute name="title">
       <string>FTP更新设置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>FTP地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ftpAddrEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>FTP端口：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="ftpPortEdit">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>用户名： </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ftpUserNameEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>密码：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ftpPassWordEdit"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>路径名： </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ftpFileNameEdit"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>上传内容</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>连接IP：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="connAddrComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>端口：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="connPortEdit">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>预  览：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fileBodyEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="systemSettingTab">
      <attribute name="title">
       <string>系统设置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>最大链接数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>主机监听端口：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>控制台前景色：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>控制台背景色：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QSpinBox" name="maxConnectionsEdit">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hostPortEdit">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>65535</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="foreColorCombo">
             <item>
              <property name="text">
               <string>浅灰</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>灰</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>白</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>黑</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>红</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>绿</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>蓝</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>黄</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>橙</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>深绿</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="backColorCombo">
             <item>
              <property name="text">
               <string>黑</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>白</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>灰</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>蓝</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>绿</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>红</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>紫</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>浅黄</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>浅蓝</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>204</width>
           <height>87</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>currentIpCheck_Clicked(bool)</slot>
 </slots>
</ui>
